tests/network/sriov.go

```diff
@@ -520,6 +523,46 @@ var _ = Describe("[Serial]SRIOV", func() {
 			})
 		})
 
+		Context("Connected to multiple SRIOV networks", func() {
+			BeforeEach(func() {
+				Expect(createSriovNetworkAttachmentDefinition(sriovnet1, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+				Expect(createSriovNetworkAttachmentDefinition(sriovnet2, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+				Expect(createSriovNetworkAttachmentDefinition(sriovnet3, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+				Expect(createSriovNetworkAttachmentDefinition(sriovnet4, util.NamespaceTestDefault, sriovConfNAD)).To(Succeed(), shouldCreateNetwork)
+			})
+
+			It("should create a virtual machine with sriov interfaces with custom MAC addresses referring the appropriate resource", func() {
+				macAddressTemplate := "de:ad:00:be:ef:%02d"
+				pciAddressTemplate := "0000:%02d:00.0"
+				sriovNetworks := []string{sriovnet1, sriovnet2, sriovnet3, sriovnet4}
+				vmi := getSriovVmi(sriovNetworks, defaultCloudInitNetworkData())
+				for i := 1; i <= 4; i++ {
+					vmi.Spec.Domain.Devices.Interfaces[i].PciAddress = fmt.Sprintf(pciAddressTemplate, i)
+					vmi.Spec.Domain.Devices.Interfaces[i].MacAddress = fmt.Sprintf(macAddressTemplate, i)
+				}
+
+				vmi = startVmi(vmi)
+				vmi = waitVmi(vmi)
+
+				By("checking KUBEVIRT_RESOURCE_NAME_<networkName> variables are defined in pod")
+				for _, name := range sriovNetworks {
+					Expect(validatePodKubevirtResourceNameByVMI(virtClient, vmi, name, sriovResourceName)).To(Succeed())
+				}
+
+				checkDefaultInterfaceInPod(vmi)
+
+				By("checking virtual machine instance has three interfaces")
+				checkInterfacesInGuest(vmi, []string{"eth0", "eth1", "eth2", "eth3", "eth4"})
```

Comment:
Why are these checks necessary? 
this test is about checking that each SR-IOV interface is configured with the correct PCI address, isn't it?
If so I would skip them.

Reply:
Why not check the usual things that are checked in this lane?

Reply:
ACK I'll make test more specific to our case.